Sources/Networking/Responses/OfferingsResponse.swift

```diff
@@ -83,6 +95,64 @@ extension OfferingsResponse.Offering.Package: Codable, Equatable {}
 extension OfferingsResponse.Offering: Codable, Equatable {}
 extension OfferingsResponse.Placements: Codable, Equatable {}
 extension OfferingsResponse.Targeting: Codable, Equatable {}
+extension OfferingsResponse.Config: Codable, Equatable {}
 extension OfferingsResponse: Codable, Equatable {}
 
 extension OfferingsResponse: HTTPResponseBody {}
+
+/*
+ Contains a set of rules that will be used when formatting a price
+ Currrently only supports overriding the currencySymbol per currency
+ */
+struct PriceFormattingRuleSet: Sendable {
+
+    // currencyCode: CurrencySymbolOverride
```

Comment:
To follow the format of the similar comment above

```suggestion
    // currencyCode -> CurrencySymbolOverride
```